### PR TITLE
chore: Add hiero-sdk-cpp and hiero-sdk-tck repos to org

### PR DESCRIPTION
**Pull request by rbarkerSL**
**Description**:

Adds hiero-sdk-cpp repository and hiero-sdk-tck repository through clowarden. Also adds the hiero-sdk-tck-maintainers team.

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -438,3 +443,15 @@ repositories:
...
+- name: hiero-sdk-cpp
+  teams:
+    tsc: maintain
+    github-maintainers: maintain
+    hiero-sdk-cpp-maintainers: maintain
+  visibility: public
+- name: hiero-sdk-tck
+  teams:
+    tsc: maintain
+    github-maintainers: maintain
```
Comment (nathanklick) [inline config.yaml:455]:
```suggestion
    github-maintainers: admin
```

File: config.yaml [hunk 2]
```diff
@@ -438,3 +443,15 @@ repositories:
     tsc: maintain
     github-maintainers: maintain
   visibility: public
+- name: hiero-sdk-cpp
+  teams:
+    tsc: maintain
+    github-maintainers: maintain
```
Comment (nathanklick) [inline config.yaml:449]:
```suggestion
    github-maintainers: admin
```

**Review by nathanklick — CHANGES_REQUESTED** [review]

File: config.yaml [hunk 3]
```diff
@@ -438,3 +443,15 @@ repositories:
...
+    hiero-sdk-cpp-maintainers: admin
+  visibility: public
+- name: hiero-sdk-tck
+  teams:
+    tsc: maintain
+    github-maintainers: maintain
```
Comment (nathanklick) [inline config.yaml:455]:
```suggestion
    github-maintainers: admin
```

File: config.yaml [hunk 4]
```diff
@@ -438,3 +443,15 @@ repositories:
...
+    hiero-sdk-tck-maintainers: admin
```
Comment (nathanklick) [inline config.yaml:456]:
```suggestion
    hiero-sdk-tck-maintainers: maintain
```

File: config.yaml [hunk 5]
```diff
@@ -438,3 +443,15 @@ repositories:
     tsc: maintain
     github-maintainers: maintain
   visibility: public
+- name: hiero-sdk-cpp
+  teams:
+    tsc: maintain
+    github-maintainers: maintain
+    hiero-sdk-cpp-maintainers: admin
```
Comment (nathanklick) [inline config.yaml:450]:
```suggestion
    hiero-sdk-cpp-maintainers: maintain
```

File: config.yaml (hunk 2 above)
Comment (nathanklick) [inline config.yaml:449]:
```suggestion
    github-maintainers: admin
```

**Review by nathanklick — CHANGES_REQUESTED** [review]

**Review by hendrikebbers — CHANGES_REQUESTED** [review]
I disagree with the suggested changes by Nathan. As described at https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization# the “maintain” role is limited in some functionalities that should be available for maintainers like delete issues. Fine with giving “GitHub-maintainers” the admin group but we should do the same for the repo maintainers for now

**Review by nathanklick — APPROVED** [review]

**Comment by nathanklick** [comment]
> I disagree with the suggested changes by Nathan. As described at https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization# the “maintain” role is limited in some functionalities that should be available for maintainers like delete issues. Fine with giving “GitHub-maintainers” the admin group but we should do the same for the repo maintainers for now

@hendrikebbers My suggested change was to give admin to the `github-maintainers` group. We should give the normal `<repo>-maintainers` the `maintain` role.

**Comment by hendrikebbers** [comment]
Agree with the first point but disagree with the second one. But I do not want to create a blocker. I will take the issue to the TSC for discussion and defining how we will do it. For now please just merge it @rbarkerSL and we can move forward.

**Pull request merged**